src/AerospikeCache.php

```diff
@@ -0,0 +1,133 @@
+<?php
+declare(strict_types=1);
```

Comment:
Please move `declare` to the first line - we have it like this in all our projects (in the other file too)

Reply:
Changed